homeassistant/__init__.py

```diff
@@ -45,6 +46,8 @@
 class HomeAssistant(object):
     """ Core class to route all communication to right components. """
 
+    arguments = None
```

Comment:
As we discussed on the chat, we need a way to get those arguments to the updater so it can call the update script with the right parameters.

I came up with two more solutions:
 1. Call ArgumentParser from updater component. Yes, will not work with other components, if the moment arrives that we ever need it, we can switch to solution 2.
 2. Have \_\_main\_\_ setup a restart method in a util package that has all parameters available

Example of 2:
```python
def restart_ha():
    […]

import homeassistant.util.restart as restart

restart.restart_ha = restart_ha
```